alma/views/templates/admin/_configure/helpers/form/15/form.tpl

```diff
@@ -278,6 +278,14 @@
                                                 class="{if isset($input.class)}{$input.class}{/if}"
                                                 value=""
                                                 {if isset($input.autocomplete) && !$input.autocomplete}autocomplete="off"{/if} />
+                                    {elseif $input.type == 'secret'}
+                                        {assign var='value_text' value=$fields_value[$input.name]}
+                                        <input type="password"
+                                            name="{$input.name}"
+                                            size="{$input.size}"
+                                            class="{if isset($input.class)}{$input.class}{/if}"
+                                            value="{if !empty($value_text)}*****{/if}"
```

Comment:
Can you use ApiHelper OBSCUR const in .TPL file?

Reply:
@Francois-Gomis 
I used the placeholder key.
Tell me what do you think ?